src/main/app/src/app/shared/utils/constants.ts

```diff
@@ -0,0 +1 @@
+export const MB_IN_BYTES= 1048576
```

Comment:
```suggestion
export const MB_IN_BYTES = 1048576
```

Reply:
As far as I can tell, that MAX_FILE_SIZE_MB is only used on the frontend, through a rest endpoint. So we don't need to change it. But I will change the comments.

Reply:
Done :-)